packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -3,23 +3,81 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import QuickLinks from './components/quick-links/QuickLinks';
 import NavItems from './components/nav-items/NavItems';
+import NavHelp from './components/nav-help/NavHelp';
+import UserProfile from './components/user-profile/UserProfile';
+import OutlineInfo from './icons/OutlineInfo';
+import UserIcon from './icons/UserIcon';
+import OutlineEllipses from './icons/OutlineEllipses';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
```

Comment:
Is this actually the layout, and not the nav? This component really needs to be renamed. Also, default proptypes shoudl be set in the examples... not in the actual component